src/query.js

```diff
@@ -519,6 +519,12 @@ class Query {
     show() {
         return this.toggle(true)
     }
+    showTr() {
```

Comment:
query here is taken from another project, https://github.com/vitmalina/mquery. It should not be modified just here. It should also be modified in the original project. Also, I do not think adding `showTr` is a good idea. It should be still within `show` possibly check if element is a TR, then do display: 'table-row'

Reply:
Well yes, everything you said is right. But so far I don't know enough about mQuery to fix it. Tried to look into but not that I'm expert on JavaScript. Maybe I can take a loog again in few days and see is there a chance for me to fix it.